docs/custom_conf.py

```diff
@@ -0,0 +1,201 @@
+import datetime
+
+# Custom configuration for the Sphinx documentation builder.
+# All configuration specific to your project should be done in this file.
+#
+# The file is included in the common conf.py configuration file.
+# You can modify any of the settings below or add any configuration that
+# is not covered by the common conf.py file.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+############################################################
+### Project information
+############################################################
+
+# Product name
+project = 'Ubuntu packaging'
+author = 'Canonical Group Ltd.'
+version = '2.0'
+
+# Uncomment if your product uses release numbers
+# release = '1.0'
+
+# The default value uses the current year as the copyright year.
+#
+# For static works, it is common to provide the year of first publication.
+# Another option is to give the first year and the current year
+# for documentation that is often changed, e.g. 2022–2023 (note the en-dash).
+#
+# A way to check a GitHub repo's creation date is to obtain a classic GitHub
+# token with 'repo' permissions here: https://github.com/settings/tokens
+# Next, use 'curl' and 'jq' to extract the date from the GitHub API's output:
+#
+# curl -H 'Authorization: token <TOKEN>' \
+#   -H 'Accept: application/vnd.github.v3.raw' \
+#   https://api.github.com/repos/canonical/<REPO> | jq '.created_at'
+
+copyright = '%s, %s' % (datetime.date.today().year, author)
+
+## Open Graph configuration - defines what is displayed in the website preview
+# The URL of the documentation output
+ogp_site_url = 'https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/'
+# The documentation website name (usually the same as the product name)
+ogp_site_name = 'Ubuntu Packaging Guide'
+# An image or logo that is used in the preview
+ogp_image = 'https://assets.ubuntu.com/v1/253da317-image-document-ubuntudocs.svg'
+
+# Update with the favicon for your product (default is the circle of friends)
+html_favicon = '.sphinx/_static/favicon.png'
+
+html_title = 'Ubuntu Packaging Guide'
+
+# (Some settings must be part of the html_context dictionary, while others
+#  are on root level. Don't move the settings.)
+html_context = {
+
+    # Change to the link to your product website (without "https://")
+    'product_page': 'packages.ubuntu.com/',
```

Comment:
nitpick: I am not sure if packages.ubuntu.com/ a good landing page for the "product". Launchpad may be better but also not ideal.

Reply:
Yeah, I wasn't sure either. Where in LP would you link? My reasoning was that packages are the desired outcome :)

Reply:
But those are community and/or dev resources. This link, in my understanding, should point to the 'product' the docs talk about. I'd say if not packages directly, then the product is Ubuntu itself.

Reply:
I think the "product" here is not packages of Ubuntu, but rather contributing (especially deb packaging related) to Ubuntu. Therefore I would argue that 2.b) makes the most sense. It also does not drop people into a random place with no further context. Any other Canonical product page also tells people what the product is and how you can use it. packages.ubuntu.com/ does neither.

But if you think that the "product" here is packages of Ubuntu then I think using 1) instead still makes sense. Launchpad is the platform where you interact with Ubuntu, its packages and its community.

Reply:
> Launchpad is the platform where you interact with Ubuntu, its packages and its community.

I think the difference is who we expect to be the main consumer of the docs. You see a community developer who would like to become an Ubuntu packager and contribute to the project. I see a customer who wants to package their own software to facilitate deployment on their installations. I suppose both are valid, but ultimately, yours makes more sense because it is a superset of mine.